app/code/Magento/Shipping/view/adminhtml/templates/view/items.phtml

```diff
@@ -13,15 +13,20 @@
             </tr>
         </thead>
         <?php $_items = $block->getShipment()->getAllItems() ?>
-        <?php $_i = 0; foreach ($_items as $_item) :
-            if (!empty($_item->getOrderItem())) :
-                if ($_item->getOrderItem()->getParentItem()) :
+        <?php $_i = 0; foreach ($_items as $_item):
+            if (!empty($_item->getOrderItem())):
+                if ($_item->getOrderItem()->getParentItem()):
                     continue;
                 endif;
                 $_i++ ?>
                 <tbody class="<?= /* @noEscape */ $_i%2 ? 'odd' : 'even' ?>">
                     <?= $block->getItemHtml($_item) ?>
                     <?= $block->getItemExtraInfoHtml($_item->getOrderItem()) ?>
+                    <?php $item = $_item->getOrderItem(); ?>
```

Comment:
Please move it before `$block->getItemExtraInfoHtml($_item->getOrderItem())` and replace this line with `$block->getItemExtraInfoHtml($item)`
It's prevent twice call for `getOrderItem` method

Reply:
hey @Den4ik . Changes are applied.